stdlib/public/stubs/LibcShims.cpp

```diff
@@ -104,12 +104,21 @@ int swift::_swift_stdlib_close(int fd) {
 static_assert(std::is_same<__swift_thread_key_t, DWORD>::value,
               "__swift_thread_key_t is not a DWORD");
 
+extern "C" void _T0s11_destroyTLSySvSgF(void *);
```

Comment:
Rather than bake in the mangling, can you use `@_silgen_name("swift_stdlib_destroyTLS_impl")` (or with a better name than impl, @moiseev any opinions there?)?

That also reminds me to update the programmers manual with a description of `@_silgen_name`...

Reply:
@milseman oh you mean, annotate the function on the swift side so it has an "extern C" name effectively?

Reply:
Exactly. As far as the name, I'm fine having the Swift function be `swift_stdlib_destroyTLS`, then you just need to come up with a different name for the shim.

Reply:
Sorry, I'm late. But you've figured it out =) Great. No unnecessary suffixes is good.

Reply:
Remember to annotate the function SWIFT_CC!